packages/core/src/internal-components/panel/panel.element.ts

```diff
@@ -32,13 +31,15 @@ export const CdsInternalPanelTagName = 'cds-internal-panel';
  * @cssprop --box-shadow
  */
 export class CdsInternalPanel extends LitElement {
+  @id() private panelId: string;
```

Comment:
I just fixed an issue with [tabbing into a scrollable panel in the modal](https://github.com/vmware/clarity/pull/5918).

We may want to double-check that isn't a problem here. Put an input or button inside a scrollable panel or card and then see if you can scroll the content with just the keyboard arrow keys. And then see if you can tab through to your button.